lib/chunk_assigner.rb

```diff
@@ -1,7 +1,7 @@
-require 'header_chunk'
-require 'paragraph_chunk'
-require 'unordered_list_chunk'
-require 'ordered_list_chunk'
+require './lib/header_chunk'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.